runtime/queries/column_timeseries.go

```diff
@@ -391,7 +400,7 @@ func getFilterFromDimensionValuesFilter(
 			var inClause = escapedName + " " + prefix + " IN ("
 			for j, iv := range dv.In {
 				if _, ok := iv.Kind.(*structpb.Value_NullValue); !ok {
-					inClause += "'" + EscapeSingleQuotes(iv.GetStringValue()) + "'"
+					inClause += "'" + escapeSingleQuotes(iv.GetStringValue()) + "'"
```

Comment:
I'm not sure `iv` is always a string value – it's just a dimension value, so could also be a boolean or number